### PR TITLE
Like button

### PR DESCRIPTION
**Pull request by onebarloop**
I wrote a function to add bookmarked artist-id's to a state array. This array renders the favorites artists list.

### CODE REVIEW

File: pages/_app.tsx [hunk 1]
```diff
@@ -1,11 +1,23 @@
...
 
 export default function App({ Component, pageProps }: AppProps) {
+  // Function for handling the favorites. Needs to be here, since the state is needed all over the app
+  const [likes, setLikes] = useState<string[]>([]);
+
+  function handleLike(id: string): void {
+    likes.includes(id)
+      ? setLikes((prev) => prev.filter((like) => like !== id))
+      : setLikes((prev) => [...prev, id]);
+  }
```
Comment (ufelgen) [inline pages/_app.tsx:13]: This is cool! I only learned this method today, so I approve ;)

File: pages/_app.tsx [hunk 2]
```diff
@@ -1,11 +1,23 @@
...
+
+  console.log(likes);
```
Comment (ufelgen) [inline pages/_app.tsx:15]: you forgot a console log here

**Review by ufelgen — COMMENTED** [review]
There's a lot of typescript in there that I don't fully understand, but it obviously works (I looked at the preview and the code) so I have nothing to complain about. I just added a comment below. 
on another note, since I looked at your preview already, I have a comment about QA: the cursor doesn't change upon hovering over the Standard View, Artist View, and Favourite buttons. It took me a while to realize that I could click those elements. This is something to keep in mind, but it's not super urgent since the target format is mobile anyway.

**Review by muhammed-basaran — COMMENTED** [review]
looks very good for me ! I think you will delete the notes later

**Comment by onebarloop** [comment]
> There's a lot of typescript in there that I don't fully understand, but it obviously works (I looked at the preview and the code) so I have nothing to complain about. I just added a comment below. on another note, since I looked at your preview already, I have a comment about QA: the cursor doesn't change upon hovering over the Standard View, Artist View, and Favourite buttons. It took me a while to realize that I could click those elements. This is something to keep in mind, but it's not super urgent since the target format is mobile anyway.

Good catch, I will change this before QA. Thanks for the review :)

**Comment by onebarloop** [comment]
> looks very good for me ! I think you will delete the notes later

Thanks! Yes, I forgot that one! 😁

**Pull request merged**